docker-compose.override.yml.example

```diff
@@ -3,3 +3,11 @@ services:
   netbox:
     ports:
       - 8000:8080
+    healthcheck:
+      # Time for which the health check can fail after the container is started.
+      # This depends mostly on the performance of your database. On the first start,
+      # when all tables need to be created the start_period should be higher than on
+      # subsequent starts. For the first start after major version upgrades of Netbox
+      # the start_period might also need to be set higher.
+      # Default value in our docker-compose.yml is 60s
+      start_period: 90s
```

Comment:
Should we also set it higher for our default config?

Reply:
I would like to leave the default as it is, docker-compose will wait to start the other containers until the start_period is over.